docs/deploy/07-bootstrap-validation.md

```diff
@@ -2,6 +2,12 @@
 
 Now that the [AKS cluster](./06-aks-cluster.md) has been deployed, the next step to validate that your cluster has been placed under a GitOps management solution, Flux in this case.
 
+## Expected results
+
+### Jump box access is validated
+
+Because the cluster is private, your cluster cannot be directly accessed locally. Therefore, you'll connect to Azure Kubernetes Service (AKS) private cluster securely using Azure Bastion's native client tunneling feature.
```

Comment:
```suggestion
Because the cluster is private, you can't directly access it from the internet. Therefore, you'll connect to Azure Kubernetes Service (AKS) private cluster securely using Azure Bastion's native client tunneling feature.
```